### PR TITLE
root: 6.32.04 -> 6.32.06

### PR DESCRIPTION
**Pull request by guitargeek**
Update ROOT 6.32 to the next patch release, which I have to use for some of my workflows.

FYI: @veprbl

### CODE REVIEW

**Review by veprbl — APPROVED** [review]
Looks great. Thank you!

File: pkgs/applications/science/misc/root/default.nix [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 stdenv.mkDerivation rec {
   pname = "root";
-  version = "6.32.04";
+  version = "6.32.06";
```
Comment (veprbl) [inline pkgs/applications/science/misc/root/default.nix:59]:
```
error: hash mismatch in fixed-output derivation '/nix/store/k1apb5rg6953jpk3hc2xivcbfvvmkqrf-root_v6.32.06.source.tar.gz.drv':
         specified: sha256-Ey8Saq59MO+8zX3NmRt62hiQrleYDvMAwWQh+dTQfqg=
            got:    sha256-P8Ay2T/oSN6lrbG0fY8KhieVIyk/7gqis81Sof+rckc=
```
you forgot to update the hash for src

Comment (guitargeek) [reply pkgs/applications/science/misc/root/default.nix:59]: Sorry, forgot to push the right hash before getting distracted by a meeting :)

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 345692` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>hepmc3</li>
    <li>ns-3</li>
    <li>python311Packages.hepmc3</li>
    <li>python312Packages.hepmc3</li>
    <li>root</li>
    <li>root.debug</li>
    <li>xfitter</li>
    <li>yoda-with-root</li>
  </ul>
</details>

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 345692` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>hepmc3</li>
    <li>ns-3</li>
    <li>python311Packages.hepmc3</li>
    <li>python312Packages.hepmc3</li>
    <li>root</li>
    <li>xfitter</li>
    <li>yoda-with-root</li>
  </ul>
</details>

**Pull request merged**